src/main/java/domain/OperatorFactory.java

```diff
@@ -0,0 +1,24 @@
+package domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperatorFactory {
+    private static final Map<String, Operator> operators = new HashMap<>();
+
+    static {
+        operators.put("+", new Plus());
+        operators.put("-", new Substraction());
+        operators.put("*", new Multiplication());
```

Comment:
```suggestion
        operators.put("*", new Operator() {
            public Number operate(int first, int second) {
                return new Number(first * second);
            }
        });
```
inner clsss가 아니라 위와 같이 구현하거나, 람다로 구현도 가능함